src/components/BulkEnrollmentPage/stepper/AddLearnersStep.jsx

```diff
@@ -72,9 +72,16 @@ const AddLearnersStep = ({
 
   const fetchData = useCallback((tableInstance = {}) => {
     const pageIndex = tableInstance.pageIndex || INITIAL_PAGE_INDEX;
+    let options = { active_only: 1, page_size: LEARNERS_PAGE_SIZE, page: pageIndex + 1 };
+
+    const { filters } = tableInstance;
+    const emailFilter = filters.find(item => item.id === 'userEmail');
+    if (emailFilter) {
+      options = { ...options, search: emailFilter.value };
+    }
     LicenseManagerApiService.fetchSubscriptionUsers(
```

Comment:
What was the decision about debouncing this call? I think it would be good to avoid a call on every keystroke.

Reply:
On reset of filtering: I double checked, and yeah the removal of all characters from the filter box does indeed reset the page to show all learners again (in that page), so looks like reset is built in


Debouncing: yeah I have not done any debouncing here yet. Agreed we should have it. Will take a look.

Reply:
added Debouncing! it works great on manually but so far I have not figure out how to fix the test that now fails to find that the 'search' param was inserted in the calls made